recipes/conda-standalone/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "conda-standalone" %}
+{% set version = "4.7.12" %}
+
+package:
+  name: conda-standalone
+  version: {{ version }}
+
+source:
+  - path: ./
```

Comment:
Is there a tarball we can use instead?

Reply:
Do you want to use https://github.com/conda/conda/archive/4.7.12.tar.gz instead?